Makefile

```
@@ -18,6 +18,9 @@ ifeq ($(V),)
 HIDE=@
 endif
 
+MIN_LIBRE	:= 2.0.1-dev
+MIN_LIBREM	:= 1.0.0-dev
```

```suggestion
LIBREM_MIN	:= 1.0.0-dev
```
perhaps this looks better?